index.js

```diff
@@ -57,3 +63,17 @@ function tryParseFloat(value) {
 	}
 	return result;
 }
+
+function getTypeFromUnit ( unit ) {
+	if ( cssAngleUnits.indexOf(unit) !== -1 ) {
+		return 'angle';
+	} else if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
```

Comment:
I knew this was "smelly" code, but I thought it covered the case in simple way. I was thinking on implementing something along using hash maps, but just thought of even "simpler" way:

```js
var reduce = require('lodash.reduce');

var type = reduce({
	angle: cssAngleUnits,
	resolution: cssResolutionUnits,
	frequency: cssFrequencyUnits,
	time: cssTimeUnits
}, function ( result, units, type ) {
	if ( units.indexOf(unit) !== -1 ) {
		return type;
	}
	return result;
}, 'length');
```

What do you think about this? Is this more performant than your suggestion? It also seems to be more readable (this is subjective opinion, of course :))